package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "devprotocol",
   "version": "0.11.0",
-  "private": true,
+  "private": false,
```

If it is not private it can be published to non registery.

```suggestion
  "private": true,
```